.github/workflows/e2e-test.yml

```diff
@@ -40,4 +26,5 @@ jobs:
       azResourceGrpAppEncrypted: ${{ needs.setup.outputs.azResourceGrpAppEncrypted }}
       ghaEnvironment: ${{ needs.setup.outputs.ghaEnvironment }}
       branchHashId: ${{ needs.setup.outputs.environmentHash }}
+      e2eCosmosDbExists: true
```

Comment:
The expectation when running this suite, is having the environment already deployed to slots, The database will already exist and be seeded.